src/Utils/SchemaExtender.php

```diff
@@ -262,9 +261,10 @@ protected static function extendValueMap(EnumType $type): array
      */
     protected static function extendPossibleTypes(UnionType $type): array
     {
-        $possibleTypes = array_map(static function ($type) {
-            return static::extendNamedType($type);
-        }, $type->getTypes());
+        $possibleTypes = [];
+        foreach ($type->getTypes() as $possibleType) {
+            $possibleTypes[] = static::extendNamedType($possibleType);
+        }
```

Comment:
```suggestion
        $possibleTypes = array_map(
           static fn ($type) => static::extendNamedType($type),
           $type->getTypes())
        );
```
in many cases here, this would be my way to go

Reply:
It is now so ;)